frontend/src/components/Footer/LinkWrapper.js

```diff
@@ -0,0 +1,11 @@
+import { Link } from "react-router-dom"
+
+const LinkWrapper = ({ to, title }) => {
```

Comment:
Like the `Instagram.js` component, `LinkWrapper` can probably be used in other places so should be moved out of the Footer folder and into the Client folder